WeakAurasOptions/RegionOptions/Text.lua

```diff
@@ -35,6 +35,15 @@ local function createOptions(id, data)
         OptionsPrivate.ResetMoverSizer();
       end,
     },
+
+    useTooltip = {
+      type = "toggle",
+      width = WeakAuras.normalWidth,
+      name = L["Tooltip on Mouseover"],
+      hidden = function() return not OptionsPrivate.Private.CanHaveTooltip(data) end,
+      order = 11
```

Comment:
That's certainly the wrong place to put it. It should be much further down.

Reply:
Can you be more specific?  This option is pretty high up in the other two regions that have it.

Reply:
You put it before the custom function that belongs to the the text edit above.

It should be below Font Flags.